Vagrantfile

```diff
@@ -71,7 +71,7 @@ $provision_script = <<SCRIPT
 set -x
 set -e
 ln -nsf /srv/zulip ~/zulip
-/usr/bin/python /srv/zulip/tools/provision.py
+/usr/bin/python /srv/zulip/tools/provision.py | sudo tee -a /var/log/zulip_provision.log
```

Comment:
This will probably need code to install `tee` on the previous line.
Also, do you need `set -o pipefail`?  See 0608e32eeb0ad52e3f9fe8c9bb3e79f8eb6d4125.

Also, does `tee` pass exit codes through correctly?